pom.xml

```diff
@@ -47,89 +62,424 @@
 		</developer>
 	</developers>
 
-	<pluginRepositories>
-		<pluginRepository>
-			<id>repo.jenkins-ci.org</id>
-			<url>http://repo.jenkins-ci.org/public/</url>
-		</pluginRepository>
-	</pluginRepositories>
+	<properties>
+		<!-- Plugin dependencies -->
+		<jenkinsCoreVersion>1.651.1</jenkinsCoreVersion>
+		<gitPluginVersion>2.4.4</gitPluginVersion>
+		<jqueryPluginVersion>1.11.2-0</jqueryPluginVersion>
+		<jodaTimeVersion>2.9.2</jodaTimeVersion>
+		<commonsLang3Version>3.4</commonsLang3Version>
+		<!-- Jenkins dependencies -->
```

Comment:
Yes, maven is a tool, but it should be used in correct way, its like use hammer for tighten the bolts.

Maven rely only on pom description, it doesn't know what exactly need for your project, and the most hard things in dependency management is transitive dependencies.

In my configuration I just fix dependency management from parent pom, because the most part of this configuration should be done by parent pom, but it is not.

I also can show you a lot of projects with dependencies problems and without proper dependency management, for example Oracle Customer Care & Billing, or open sourced - rundeck.

But it doesn't mean that you need to repeat their mistakes.

Just take a look on convergence report and you'll see how commons-io from 2.4 version omitted to 1.4 and much more.

P.S. mvn project-info-reports:dependency-convergence